<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidRender</class>
 <widget class="QDockWidget" name="DockWidRender">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>300</height>
   </rect>
  </property>
  <property name="floating">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Rendering properties</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Vertical range</string>
       </attribute>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Vertical range</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>70</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Vertical shift</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_3">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>30</y>
          <width>47</width>
          <height>23</height>
         </rect>
        </property>
        <property name="minimum">
         <number>20</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_4">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>70</y>
          <width>47</width>
          <height>23</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
       <widget class="QDial" name="dial">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>20</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="minimum">
         <number>20</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
       <widget class="QDial" name="dial_2">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>60</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Lines</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Vertical lines</string>
          </property>
          <widget class="QCheckBox" name="checkBox">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>30</y>
             <width>101</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Central line</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QSpinBox" name="spinBox">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>60</y>
             <width>71</width>
             <height>23</height>
            </rect>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>64</number>
           </property>
           <property name="value">
            <number>4</number>
           </property>
          </widget>
          <widget class="QCheckBox" name="checkBox_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>60</y>
             <width>161</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Beat lines; measure:</string>
           </property>
          </widget>
          <widget class="QComboBox" name="comboBox_2">
           <property name="geometry">
            <rect>
             <x>260</x>
             <y>60</y>
             <width>71</width>
             <height>22</height>
            </rect>
           </property>
           <property name="currentText">
            <string>1</string>
           </property>
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>16</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>32</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>64</string>
            </property>
           </item>
          </widget>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Horizontal lines</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="4" column="1">
            <widget class="QWidget" name="widget" native="true">
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QPushButton" name="pushButton">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>Set color...</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QSpinBox" name="spinBox_8">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>50</height>
              </size>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>96</number>
             </property>
             <property name="value">
              <number>12</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="checkBox_8">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>On</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="comboBox">
             <item>
              <property name="text">
               <string>1 - One line every n semitones</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2 - One line every semitone, one stronger line every n semitones</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3 - Keyboard</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4 - Keyboard (inverse)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5 - Staff</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QSpinBox" name="spinBox_9"/>
           </item>
           <item row="3" column="0">
            <widget class="QCheckBox" name="checkBox_9">
             <property name="text">
              <string>Half shift</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tracks rendering</string>
       </attribute>
       <widget class="QCheckBox" name="checkBox_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>221</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Render playing notes separately</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Render tracks separately</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_2">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>20</y>
          <width>47</width>
          <height>23</height>
         </rect>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>20</y>
          <width>31</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Blur:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_5">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>20</y>
          <width>47</width>
          <height>23</height>
         </rect>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>231</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Render moving notes separately</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>80</y>
          <width>31</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Blur:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_6">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>80</y>
          <width>47</width>
          <height>23</height>
         </rect>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_7">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>80</y>
          <width>47</width>
          <height>23</height>
         </rect>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Time</string>
       </attribute>
       <widget class="QCheckBox" name="checkBox_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>231</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Extra zoom time at the beginning</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>211</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Extra zoom time at the end</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Video codec</string>
       </attribute>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>221</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Video output FourCC codec code:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>20</y>
          <width>51</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>X264</string>
        </property>
        <property name="maxLength">
         <number>4</number>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBox_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dial_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
